app/controllers/api/v2/hosts_controller.rb

```diff
@@ -62,7 +65,10 @@ def show
           param :owner_type, Host::Base::OWNER_TYPES, :desc => N_("Host's owner type")
           param :puppet_ca_proxy_id, :number
           param :image_id, :number
-          param :host_parameters_attributes, Array
+          param :host_parameters_attributes, Array, :desc => N_("Host's parameters (array or indexed hash)") do
+            param :name, String, :desc => "Name of the parameter", :required => true
+            param :value, String, :desc => "Parameter value", :required => true
```

Comment:
> ditto

Right, copy pasted the doco into the V1 controller as well.

-- 
Later,
 Lukas #lzap Zapletal